### PR TITLE
Implement WP coding standards

### PR DESCRIPTION
**Pull request by benbalter**
Fixes https://github.com/benbalter/wp-document-revisions/issues/92.

### CODE REVIEW

**Comment by benbalter** [comment]
@jeremyfelt I believe this should be ready for your review. I chose to ignore a handful of the styles that required major rewrites, but believe otherwise, both unit tests and style tests should be passing.

File: phpcs.ruleset.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Plugins">
+	<description>Generally-applicable sniffs for WordPress plugins</description>
+
+	<rule ref="WordPress-Core">
+		<exclude name="Squiz.Commenting.FunctionComment.ParamCommentFullStop" />
+		<exclude name="Squiz.Commenting.InlineComment.InvalidEndChar" />
+		<exclude name="Generic.PHP.NoSilencedErrors.Discouraged" />
+		<exclude name="WordPress.CSRF.NonceVerification.NoNonceVerification" />
```
Comment (aaronjorbin) [inline phpcs.ruleset.xml]: `WordPress.CSRF.NonceVerification.NoNonceVerification` is a pretty good rule to leave in place.

File: includes/class-wp-document-revisions.php [hunk 2]
```diff
@@ -0,0 +1,2118 @@
...
+ * Main class for WP Document Revisions
+ *
+ * @since 3.0.0
+ * @package WP_Document_Revisions
+ */
+
+/**
+ * Main WP_Document_Revisions class
+ */
+class WP_Document_Revisions extends HTTP_WebDAV_Server {
```
Comment (aaronjorbin) [inline includes/class-wp-document-revisions.php]: Renaming this class is a breaking change.  If intentional, it should be documented as such.

**Comment by jeremyfelt** [comment]
Totally missed this notification. Excellent stuff. Happy to take on some of the larger rewrites for skipped rules if necessary. I'll review in detail in the next day or two!

File: phpcs.ruleset.xml (hunk 1 above)
Comment (jeremyfelt) [inline phpcs.ruleset.xml]: Agreed. This might be a good candidate to skip for now and address in another PR later.

Comment (benbalter) [inline phpcs.ruleset.xml]: Completely agree. My thinking was to make less-invasive style changes, and then to make targeted, more invasive changes in a subsequent pass.

**Comment by benbalter** [comment]
This diff is getting big. Manually confirmed things were working as expected locally, going to merge, and will follow up in a subsequent pass.

**Pull request merged**